BioArchLinux/minimap2/PKGBUILD

```diff
@@ -3,23 +3,46 @@
 
 pkgname=minimap2
 pkgver=2.26
-pkgrel=1
+pkgrel=2
 pkgdesc='A versatile pairwise aligner for genomic and spliced nucleotide sequences. https://doi.org/10.1093/bioinformatics/bty191'
 arch=('x86_64')
 url="https://github.com/lh3/${pkgname}"
 license=('MIT')
-depends=(zlib)
-provides=()
+depends=('zlib' 'glibc')
+provides=('minimap2-lite' 'sdust')
 source=("${pkgver}.tar.gz::$url/archive/refs/tags/v$pkgver.tar.gz")
```

Comment:
`$pkgname-$pkgver.tar.gz` would be a unique name for the tarball.

Reply:
Done